xml/XSLT/throttle-include.xsl

```diff
@@ -0,0 +1,135 @@
+      <xsl:for-each select="@*"> 
+        <tr>
+          <td><xsl:value-of select="name()"/></td>
+          <td>
+          <xsl:choose>
+            <xsl:when test='name() ="displaySpeedSlider"'>              
+              <xsl:choose>
+                <xsl:when test='. = 0'>0 (Percentage)</xsl:when>
+                <xsl:when test='. = 1'>1 (Speed Steps)</xsl:when>
+                <xsl:when test='. = 2'>2 (Shunting)</xsl:when>
+                <xsl:otherwise><xsl:value-of select="."/></xsl:otherwise>
+              </xsl:choose>
+            </xsl:when>
+            <xsl:otherwise>        
+              <xsl:value-of select="."/>
+            </xsl:otherwise>
+          </xsl:choose>
+          </td>
+        </tr>
+       </xsl:for-each>
+    </table>
+  </xsl:template>
+
+
+  <!-- Display function panel subwindow -->
+  <xsl:template match="FunctionPanel">
+    <h4>Function Panel  <xsl:call-template name="window"/></h4>
+    <table border="border-width:thin" width="100%">
+      <tr>
+        <th>Button ID</th>
+        <th>Text</th>
+        <th>Lockable</th>
+        <th>Visible</th>
+        <th>fontSize</th>
+        <th>Icon</th>
+      </tr>
+      <xsl:apply-templates/>
+    </table>
+  </xsl:template>
+
+  <xsl:template match="FunctionButton">
+    <tr>
+     <td style="text-align:center"><xsl:value-of select="@id"/></td>
+     <td><xsl:value-of select="@text"/></td>
+     <td style="text-align:center"><xsl:value-of select="@isLockable"/></td>
+     <td style="text-align:center"><xsl:value-of select="@isVisible"/></td>
+     <td style="text-align:center"><xsl:value-of select="@fontSize"/></td>
+     <td>ImageSize: <xsl:value-of select="@buttonImageSize"/> 
+       <xsl:if test='@iconPath != ""'><br/>"Off" Icon: <xsl:value-of select="@iconPath"/></xsl:if> 
+       <xsl:if test='@selectedIconPath != ""'><br/>"On" Icon: <xsl:value-of select="@selectedIconPath" /></xsl:if>
+     </td>
+   </tr>
+  </xsl:template>
+
+  <!-- Display address panel subwindow -->
+  <xsl:template match="AddressPanel">
+    <h4>Address Panel   <xsl:call-template name="window"/></h4>
+    <table border="border-width:thin" width="33%">
+      <tr>
+        <th>Loco Number</th>
+        <th>Attributes</th>
+      </tr>
+      <xsl:apply-templates/>
+    </table>
+  </xsl:template>
+
+  <xsl:template match="locoaddress">
+    <xsl:if test='number > 0'>
+      <tr>
+        <td>
+          <xsl:if test='number > 0'>
+            <xsl:value-of select="number"/>
+          </xsl:if>
+        </td>
+        <td>
+          <xsl:if test='dcclocoaddress/@number > 0'>DCC</xsl:if>
+          <xsl:if test='dcclocoaddress/@longaddress = "yes"'>[Long]</xsl:if>
+        </td>
+      </tr>
+    </xsl:if>
+  </xsl:template>
+
+  <!-- Display speed panel subwindow -->
+  <xsl:template match="SpeedPanel">
+    <h4>Speed Panel     <xsl:call-template name="window"/></h4>
+    <xsl:apply-templates/>
+  </xsl:template>
+
+  <!-- Display Jynstrument panel subwindow -->
```

Comment:
Since the Jynstrument sections are different for throttle-config and throttle-layout-config, they should be in the corresponding config file.

Reply:
@dsand47 Done.  Works with your examples from #10831 and others I have tried.

Reply:
Panel type Jynstrument put in include file since it can appear in both throttle layouts and configs.  Toolbar type Jynstrument put in throttle-layout-config since it only appears there.